deploy/hubble/values.yaml

```diff
@@ -55,6 +55,6 @@ affinity: {}
 
 persistentStorage:
   size: 100Gi
-  volumeID: aws://us-east-1a/vol-000677922f895379c # example: aws://us-east-1c/vol-aabbcc123
+  volumeID: WILL_BE_REPLACED # example: aws://us-east-1c/vol-aabbcc123
```

Comment:
We can keep prod here as default and override only for stage

Reply:
sure, give me sec. Need to create EBS for prod first